### ISSUE TITLE
Output Table Issues - Intensity vs. Unique Intensity vs. Total Intensity

### ISSUE THREAD

**Issue by mamccabe** [Nesvilab/philosopher#273]
### Discussed in https://github.com/Nesvilab/philosopher/discussions/271

<div type='discussions-op-text'>

<sup>Originally posted by **mamccabe** September  1, 2021</sup>
Hello,

I have been using MSFragger for processing large proteomics datasets but I believe there is a bug in how the intensities are reported, at least in the combined protein report. 

The issue is that the relative column values apparently do not match their descriptions. In the second sheet of the attached Excel file, you can see 3 columns where the unique, total, and standard intensity values are compared for a single sample. Based on the descriptions in the wiki, you would expect that Total Intensity (all matching peptides, regardless of uniqueness) would always be the same or higher than the unique intensity (only peptides unique to that protein). However, in the third line down (as well as in many other cases), you can see that the unique intensity is higher than the total for that protein. Additionally, one would assume that "Intensity" (peptides assigned to a protein using the razor rule + unique peptides) should always be higher or the same compared to the unique intensities, but there are cases where the "intensity" is lower than the unique peptide value. 

In summary, there are no consistent relationships between the three columns for intensity which logically match the descriptions provided on the wiki. I have checked and this is not the case for spectral matches (where these comparison assumptions do align with the reported data), only for the intensity values.

Thank you!

Max

[Philosopher Support Upload.xlsx](https://github.com/Nesvilab/philosopher/files/7092843/Philosopher.Support.Upload.xlsx)
</div>

**Comment by prvst** [comment]
I'm closing here because Sarah already replied to you via Discussions.
